releasenotes/index.rst

```diff
@@ -13,16 +13,17 @@ see the Release Notes for it.
 Current Stable
 ==============
 
-* :ref:`Dojo Toolkit v1.11 Release Notes <releasenotes/1.11>`
+* :ref:`Dojo Toolkit v1.12 Release Notes <releasenotes/1.12>`
 
 Current Development
 ===================
 
-* :ref:`Dojo Toolkit v1.12 Release Notes <releasenotes/1.12>`
+* :ref:`Dojo Toolkit v1.13 Release Notes <releasenotes/1.13>`
```

Comment:
This looks good, let's remember to add this file also and then we're good to go.